synop2bufr/__init__.py

```diff
@@ -1337,6 +1405,23 @@ def transform(data: str, metadata: str, year: int,
 
             # now identifier based on WSI and observation date as identifier
             isodate = message.get_datetime().strftime('%Y%m%dT%H%M%S')
+
+            # Write message to CSV file
```

Comment:
We do not want this is synop2bufr as part of the API, there should be no file system access,